test/metabase/events/audit_log_test.clj

```diff
@@ -288,25 +288,23 @@
 (deftest metric-update-event-test
   (testing :metric-update
     (t2.with-temp/with-temp [Metric metric {:table_id (mt/id :venues)}]
-      (mt/with-model-cleanup [:model/AuditLog]
-        (mt/with-test-user :rasta
-         (let [event (-> (assoc metric
-                                :actor_id         (mt/user->id :rasta)
-                                :revision_message "update this mofo")
-                         ;; doing this specifically to ensure :actor_id is utilized
-                         (dissoc :creator_id))]
-           (is (= event
-                  (events/publish-event! :event/metric-update event))))
-         (is (= {:topic       :metric-update
-                 :user_id     (mt/user->id :rasta)
-                 :model       "Metric"
-                 :model_id    (:id metric)
-                 :details     {:name             (:name metric)
-                               :description      (:description metric)
-                               :revision_message "update this mofo"
-                               :database_id (mt/id)
-                               :table_id    (mt/id :venues)}}
-                (event "metric-update" (:id metric)))))))))
+      (mt/with-test-user :rasta
+        (let [event (-> (assoc metric
+                               :actor_id         (mt/user->id :rasta)
+                               :revision_message "update this mofo")
+                        ;; doing this specifically to ensure :actor_id is utilized
```

Comment:
I am not sure that I understand how `:actor_id` is different or important in here, so please scrutinize this test carefully 🙏

Reply:
It shouldn't be needed here anymore